spec/unit/ssl/state_machine_spec.rb

```diff
@@ -496,6 +548,8 @@ def expect_lockfile_to_contain(pid)
         allow(cert_provider).to receive(:load_private_key).and_return(private_key)
         allow(cert_provider).to receive(:load_client_cert).and_return(cert_fixture('tampered-cert.pem'))
 
+        ssl_context = Puppet::SSL::SSLContext.new(cacerts: [cacert], crls: [crl])
+        state = Puppet::SSL::StateMachine::NeedKey.new(machine, ssl_context)
```

Comment:
How did this test pass before this change?

Reply:
This constructs the `SSLContext` and `NeedKey` state the same way that it did prior, with only the root CA and CRL.